predicators/envs/spot_env.py

```diff
@@ -696,6 +707,44 @@ def _reachable_classifier(state: State, objects: Sequence[Object]) -> bool:
     return is_xy_near and is_yaw_near
 
 
+def _blocking_classifier(state: State, objects: Sequence[Object]) -> bool:
```

Comment:
I think this is a good initial implementation, but not general. Can we leave a comment describing that this isn't the real right way to do this (the right way would be to do TAMP or implement something that uses the collision bodies to see if there is a pose that we can find that isn't constrained)?